source/org.chromium.net/cronet-embedded/Transforms/Metadata.Namespaces.xml

```diff
@@ -76,5 +76,41 @@
         >
         Xamarin.Chromium.J
     </attr>    
+    <attr
+        path="/api/package[@name='org.chromium.base']"
+        name="managedName"
+        >
+        Xamarin.Chromium.Base
+    </attr>    
+    <attr
+        path="/api/package[@name='org.chromium.base.supplier']"
+        name="managedName"
+        >
+        Xamarin.Chromium.Base.Supplier
+    </attr>    
+    <attr
+        path="/api/package[@name='org.chromium.build']"
+        name="managedName"
+        >
+        Xamarin.Chromium.Build
+    </attr>    
+    <attr
+        path="/api/package[@name='org.chromium']"
+        name="managedName"
+        >
+        Xamarin.Chromium
+    </attr>    
+    <attr
+        path="/api/package[@name='org.chromium.base.natives']"
+        name="managedName"
+        >
+        Xamarin.Chromium.Base.Natives
+    </attr>    
+    <attr
+        path="/api/package[@name='org.chromium.native_test']"
+        name="managedName"
+        >
+        Xamarin.Chromium.Native_Test
```

Comment:
Should this be `NativeTest`?  I don't have an opinion.

Reply:
I was scratching my head. Also lack of opinion. So i said, let's try it this way. From my experience, such namespaces will be removed soon anyway.